src/ds/d400/d400-motion.cpp

```diff
@@ -41,6 +41,19 @@ namespace librealsense
         return _ds_motion_common->get_motion_intrinsics(stream);
     }
 
+    bool d400_motion_base::is_imu_high_accuracy() const
+    {
+        // D400 FW 5.16 and above use 32 bits in the struct, instead of 16.
+        return _fw_version >= firmware_version( 5, 16, 0, 0 ); 
+    }
+
+    double d400_motion_base::get_gyro_default_scale() const
+    {
+        // FW scale in the HID feature report was 10 up to 5.16, changed to 1000 to support gyro sensitivity option.
```

Comment:
Please clarify that 5.16 is a FW version

Reply:
Done